README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Python
-==========================================
+Confluent's Python Client for Apache Kafka<sup>TM</sup>
```

Comment:
rule of thumb from http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html "Capitalize all words in titles of publications and documents, except a, an, the, at, by, for, in, of, on, to, up, and, as, but, or, and nor."